lib/pbench/server/auth/auth.py

```diff
@@ -128,7 +186,9 @@ def verify_auth(auth_token):
         return None
 
     @staticmethod
-    def verify_third_party_token(auth_token: str, oidc_client: OpenIDClient) -> bool:
+    def verify_third_party_token(
+        auth_token: str, oidc_client: OpenIDClient, algorithms: List[str]
```

Comment:
Should it be `algorithms: Optional[list[str]]` with an implicit default?? If the real auth always uses `["HS256"]` do we really need to pass it every time instead of assuming it? Or since the list is hardcoded, so we really need the argument at all?